content/blog/news/gsoc-2020-cloud-plugin.md

```diff
@@ -0,0 +1,155 @@
+
+3. The plugin itself then puts together two separate widgets:
+
+    * [CloudManagerTreeWidget](https://github.com/AcademySoftwareFoundation/OpenCue/blob/d4a2eca93d271dacb13ca2aaba9b531642c1dc6b/cuegui/cuegui/CloudManagerWidget.py#L63): 
+    This implements the [AbstractTreeWidget](https://github.com/AcademySoftwareFoundation/OpenCue/blob/526efee2fbf8c442a8e9d631e1806854463c4301/cuegui/cuegui/AbstractTreeWidget.py) class which provides very useful methods to handle the way data has 
+    to be displayed in the widget.
+    
+    * [CloudGroupCreateDialog](https://github.com/AcademySoftwareFoundation/OpenCue/blob/d4a2eca93d271dacb13ca2aaba9b531642c1dc6b/cuegui/cuegui/CloudGroupDialog.py#L28):
+    Which is used outside of the tree view and launched when the `Add Cloud Group` button is clicked on the plugin.
+    
+     
+The interface uses the main `CloudManager` api, mentioned above, as the entry point to then get all cloud groups
+associated with all the registered cloud providers.
+
+
+## Cloud plugin setup
+
+### Google compute engine setup
+
+{{% alert title="Note" color="info"%}}My deployment included setting up Cuebot on GCE{{% /alert %}}
+
+Listing the minimum requirements for the cloud plugin to work with GCE:
+
+1. GCP project with Compute Engine API and Cloud Logging API enabled.
+
+2. Setup [Application Default Credentials](https://cloud.google.com/docs/authentication/production) and make sure the google python api client (which is a requirement for this plugin)
+ has access to it.
+
+3. Use the `cloud_plugin_resources.yaml` under the `cuegui/config` directory to setup your GCP project name and
+the zone.
+
+4. At least one instance template setup on the cloud: I was using a RQD Blender docker image uploaded as an
+[Instance Template](https://cloud.google.com/compute/docs/instance-templates) with its appropriate VM type.
+
+
+### Cloud plugin interface
+
+The Cloud plugin should be available to launch from the menu bar by navigating: 
+**Views/Plugins -> CueCommander -> Cloud Plugin**
+
+1. The cloud plugin interface consists mainly of two components:
+    1. Main window which shows the list of cloud groups available with all registered providers.
+    2. A cloud group addition dialog that can be used to create a cloud group from the interface itself.
+    
+### Cloud Plugin functionality
+ 
+1. To start create a cloud pool group, use the `Add Cloud Group` button to launch the 
+dialog that will allow you to create a cloud group.
+
+![Add Cloud Group Dialog](/blog/images/AddCloudGroupDialog.png)
+
+2. If the API request is successful you should now see an entry in the Cloud Plugin's main interface.
+
+![After cloud group addition](/blog/images/MainInterfacePostAddition.png)
+
+3. The right-click content menu has two additional functionality: 
+    1. Resizing (scaling up/down) the number of instances in a group.
+    2. Deleting the group altogether.
+
+![Cloud Plugin Context Menu](/blog/images/CloudPluginContextMenu.png)
+
```

Comment:
I would probably add a list item here about what else is needed to use the instances for a job, which is hopefully nothing. Something like -- once the instances start and RQD launches, you should see the host appear in the Monitor Hosts plugin, after which OpenCue will be able to send jobs to that host.